### ISSUE TITLE
Custom time signatures broken in 'Palettes' and 'Master Palette'

### ISSUE THREAD

**Issue by Tantacrul** [musescore/MuseScore#10404]
**Describe the bug**
See video

https://user-images.githubusercontent.com/21022311/151659948-af5235c7-507b-461b-bdfd-9138b92bd625.mov

**Comment by Tantacrul** [comment]
@bkunda - you may have logged this already. I'm also aware there's a redesign of part of the custom time signatures modal too, which you may want to link to here.

**Comment by bkunda** [comment]
Hey @Tantacrul, yes this issue has been raised in #10372.
There is also some broken functionality in the create time signature dialog, which I raised in #9618. @martin-danhier is already onto fixing this.
And the design improvements are in https://github.com/musescore/MuseScore/issues/9620 (Again, with @martin-danhier on the case).  
But I have not yet discovered the problem with the master palette (from your video from 01:38).  I think it would be good if we rescoped the present issue to focus on addressing just the problem of not being able to drag elements from the master palette to the main palettes 🙂. 
Cheers!
Bradley
